micall_docker.py

```diff
@@ -985,9 +982,8 @@ def link_samples(
         run_path: str,
         data_path: str,
         is_denovo: bool,
-        fastq1s: Optional[Iterable[str]],
-        fastq2s: Optional[Iterable[str]],
-):
+        fastq1s: typing.Sequence[str] = None,
```

Comment:
Should this not be `Optional[Sequence[str]]`?

Reply:
I thought the `= None` implied the `Optional`. PyCharm wasn't complaining, but I haven't checked the rules or tried mypy.

Reply:
Huh, so it does. Today I Learned ([docs, for future reference](https://docs.python.org/3/library/typing.html#typing.get_type_hints))